js/calculate.js

```diff
@@ -0,0 +1,127 @@
+const operate = require('./operate');
+
+function calculate(input, calcObject) {
+  let { operandOne = null, operandTwo = null, displayValue = null, operator = null } = calcObject;
+  if (['0', '1', '2', '3', '4', '5', '6', '7', '8', '9'].includes(input)) {
```

Comment:
This is nice, but can you refactor the code to use `switch-case`? see https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Statements/switch

Reply:
Done